Add Trino Fest recap blog post (#459)
trinodb/trino.io: _posts/2023-06-20-trino-fest-2023-recap.md

---
layout: post
title: "Trino Fest 2023 recap"
author: "Manfred Moser, Cole Bowden"
excerpt_separator: <!--more-->
image: /assets/blog/trino-fest-2023/trino-fest.png
---

Last week we held Trino Fest, and we did not find time to chill at the
lakehouse. Despite the theme of a summercamp for lakehouse analytics, the fest

> kept us all busy. Great demos, amazing announcements, new plugins, and use cases

Reviewer: I feel like this is a bit of a negative tone for starting things off. Maybe re-order to, "and it kept us all so busy, we forgot to spend time chilling by the lakehouse!" That feels a little more cheeky and fun IMO.

Author: Done

reached our active audience. Thanks go to our event host and organizer

> [Starburst](https://www.starburst.io/), our sponsors
> [AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), and many
> well-prepared speakers, and our great live audience. Now you get a chance to

Reviewer: ```suggestion
[Starburst](https://www.starburst.io/), to our sponsors
[AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), to our many
well-prepared speakers, and to our great live audience. Now you get a chance to
```

catch up on anything you missed.

<!--more-->

<div class="container">
  <div class="row">
    <div class="col-sm">
      <a href="https://www.starburst.io/">
        <img src="{{site.url}}/assets/images/logos/starburst-small.png" title=" Starburst, event host and organizer ">
      </a>
    </div>
    <div class="col-sm">
      <a href="https://www.alluxio.io/">
        <img src="{{site.url}}/assets/images/logos/alluxio-small.png" title="Alluxio, event sponsor">
      </a>
    </div>
    <div class="col-sm">
      <a href="https://aws.amazon.com/">
        <img src="{{site.url}}/assets/images/logos/aws-small.png" title="AWS, event sponsor">
      </a>
    </div>
  </div>
</div>

> In the weeks running up to the event we had numerous blog posts going out, and

Reviewer: ```suggestion
In the weeks leading up to the event, we published numerous blog posts and
```

racked up great interest in the Trino community and beyond. Over 1100
registrations blew away our numbers from last year. More importantly, during the

> two half days of the event we had up to over 560 attendees watching live and

Reviewer: ```suggestion
two half-days of the event, we had over 560 attendees watching live and
```

Reviewer: we can do "up to" or "over" but not both 😛

Author: hahaha

participating in the busy chat.

## Sessions

> And now all those of you, who could not attend every session, or missed out on

Reviewer: ```suggestion
If you could not attend every session, or if you missed out on
```

> attending completely, have a chance to learn from the presentations and the

Reviewer: ```suggestion
attending completely, then we've got great news for you! You still 
have a chance to learn from the presentations and the
```

  Miguel Filipe and Jonas Irgens Kylling from [Dune](https://dune.com/).
* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Data Mesh
  implementation using Hive views](https://youtu.be/ZgcVtPFkKHM) presented by
  Alejandro Rojas from [Comcast](https://comcast.github.io/).
* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Inspecting
  Trino on ice](https://youtu.be/PSGuAMVc6-w) presented by Kevin Liu from
  [Stripe](https://stripe.com/).
* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Trino
  optimization with distributed caching on Data
  Lake](https://youtu.be/oK1A5U1WzFc) presented by Hope Wang and Beinan Wang
  from [Alluxio](https://www.alluxio.io/).
* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Starburst
  Galaxy: A romance of many architectures](https://youtu.be/K3AlAWB-Gmg)
  presented by Benjamin Jeter from [Datto](https://www.datto.com/).
* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>FugueSQL,
  Interoperable Python and Trino for interactive
  workloads](https://youtu.be/aKhI1Phfn-o) presented by [Kevin
  Kho](https://www.linkedin.com/in/kvnkho/).

## Next up

> With this first recap we are sharing all the video recordings with you all. Next
> up are individual blog post for each session that include further useful
> material:

Reviewer: ```suggestion
This first recap is sharing all the video recordings with you all if you can't wait.
But stay tuned, because we'll also be publishing individual blog posts and recaps
for each session, and they'll include additional useful info:
```

> 

Reviewer: Include a line about a summary or recap of the main talking points.

> * Presentation file to look at your own pace

Reviewer: ```suggestion
* Slide decks for you to browse on your own
```

> * Video recording

Reviewer: Delete this? We're already linking the video recordings here.

* Interesting and fun quotes from the speakers and audience
* Notes and impressions from the audience and event hosts
* Questions and answer during the event
* Links to further documentation, tutorials, and other resources

> Putting all this together will take us a while, so keep an eye out on our

Reviewer: ```suggestion
We'll be rolling out recap posts for a few talks each week, so keep an eye out on our
```
I think "it's gonna take a while" makes it sound like we're incapable of doing work faster. We _are_ capable, we're just choosing to spread the content out to drive more engagement/views on LinkedIn and send a larger volume of traffic to the website over time, rather than in one big burst.

[community chat]({{site.url}}/slack.html) or the website for updates.

> At the same time we are already marching ahead with planning towards our next

Reviewer: ```suggestion
At the same time, we are already marching ahead and planning our next major
```

image: /assets/blog/trino-fest-2023/trino-fest.png
---

> Last week we held Trino Fest, and it kept us all so busy, we forgot to spend

Reviewer 2: Unrelated changes for "Remove invalid AWS logos"

Author: Well ... the logo is used in the post and they are separate commits

Reviewer 2: This commit ("Remove invalid AWS logos") has changes that have nothing to do with removing the logo. Why did this paragraph change? If you look below, there are some other paragraphs with grammar changes that are unrelated to removing the logo.
